scripts/deblur

```diff
@@ -188,55 +276,65 @@ def multiple_seq_alignment(seqs_fp, output_fp):
-                                      "sample ID from sequence ID")
-def build_biom_table(seqs_fp, uc_fp, output_biom_fp, output_rep_set_fp, delim):
-    """Generate a BIOM table"""
-    deblur_clrs, table = generate_biom_table(seqs_fp=seqs_fp,
-                                             uc_fp=uc_fp,
-                                             delim=delim)
-    # write BIOM table to file
-    if table.is_empty():
-        raise ValueError(
-            "Attempting to write an empty BIOM table.")
-    with biom_open(output_biom_fp, 'w') as f:
-        if HAVE_H5PY:
-            table.to_hdf5(h5grp=f, generated_by="deblur")
-        else:
-            table.to_json(direct_io=f, generated_by="deblur")
+                                file_okay=True), help='name of output biom table')
+@click.argument('file_type', required=False, type=str, default='.fasta.trim.derep.no_artifacts.msa.deblur.no_chimeras',
+                show_default=True, help='ending of files to be added to the biom table')
+@click.option('--log-level', required=False, type=int, default=2,
+              show_default=True, help="Level of messages for log file"
+              "(range 1-debug to 5-critical")
+@click.option('--log-file', required=False, type=click.Path(resolve_path=True, readable=True, exists=False,
+              dir_okay=True), default='deblur.log',
+              show_default=True, help="log file name")
+def build_biom_table(seqs_fp, output_biom_fp, file_type, log_level, log_file):
+    """Generate a BIOM table from a directory of chimera removed fasta files
+    Parameters
+    ----------
+    seqs_fp : str
+      the path to the directory containing the chimera removed fasta files
+    output_biom_fp : str
+      the path where to save the output biom table file ('final.biom')
+    file_type : str
+      the files type to add to the table
+      (default='.trim.derep.no_artifacts.msa.deblur.no_chimeras',
+      can be '.fasta' or '.fa' if needed)
+    """
+    start_log(level=log_level * 10, filename=log_file)
+    output_filename = 'final.biom'
+    output_fp = join(output_biom_fp, output_filename)
 
-    # write representative FASTA sequences to file
-    with open(output_rep_set_fp, 'w') as f:
-        for s in deblur_clrs:
-            f.write(">%s\n%s\n" % (deblur_clrs[s][0], s))
+    samples = get_files_for_table(seqs_fp, file_type)
+    create_otu_table(output_fp, samples)
 
 
 # LAUNCH FULL DEBLUR PIPELINE COMMAND
 @deblur_cmds.command()
 @click.option('--seqs-fp', required=True,
               type=click.Path(resolve_path=True, readable=True, exists=True,
-                              file_okay=True),
-              help="Demultiplexed FASTA file including all samples")
+                              file_okay=True, dir_okay=True),
+              help="Demultiplexed FASTA file including all samples or a directory of .fasta files (requires --no-split)")
+@click.option('--no-split', required=False, type=bool, default=False, is_flag=True,
```

Comment:
Do you really need this flag? What I would is check the type of the input using `os.path.isdir`. If it is a directory, then you assume that it is a directory of fasta files, if not, it is a fasta file that you need to split.